scully.scully-docs.config.ts

```diff
@@ -64,6 +64,7 @@ async function createConfig(): Promise<ScullyConfig> {
     outDir: './dist/static/doc-sites',
     distFolder: './dist/apps/scully-docs',
     defaultPostRenderers,
+    // extraRoutes: ['/ngconf'],
```

Comment:
nitpicking but this can be removed as I believe

Reply:
yep. will do that in one of the next pr's